<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Tool Box</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>30</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonRectangle">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>PaintIcons/rect.png</normaloff>PaintIcons/rect.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonEllipse">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>PaintIcons/ellipse.png</normaloff>PaintIcons/ellipse.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonLine">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>PaintIcons/line.png</normaloff>PaintIcons/line.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRoundRectangle">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>PaintIcons/roundrectangle.png</normaloff>PaintIcons/roundrectangle.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPolygon">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>PaintIcons/polygon.png</normaloff>PaintIcons/polygon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPen">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>PaintIcons/icons8-pen-40.png</normaloff>PaintIcons/icons8-pen-40.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose_Window"/>
    <addaction name="actionOpen_New_Window"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose_Window">
   <property name="text">
    <string>Close Window</string>
   </property>
  </action>
  <action name="actionOpen_New_Window">
   <property name="text">
    <string>Open New Window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
